src/core/server/plugins/plugins_service.test.ts

```diff
@@ -244,10 +244,10 @@ describe('PluginsService', () => {
             "Plugin \\"explicitly-disabled-plugin\\" is disabled.",
           ],
           Array [
-            "Plugin \\"plugin-with-missing-required-deps\\" has been disabled since some of its direct or transitive dependencies are missing or disabled.",
+            "Plugin \\"plugin-with-missing-required-deps\\" has been disabled since the following direct or transitive dependencies are missing or disabled: [\\"missing-plugin\\"]",
           ],
           Array [
-            "Plugin \\"plugin-with-disabled-transitive-dep\\" has been disabled since some of its direct or transitive dependencies are missing or disabled.",
+            "Plugin \\"plugin-with-disabled-transitive-dep\\" has been disabled since the following direct or transitive dependencies are missing or disabled: [\\"another-explicitly-disabled-plugin\\"]",
```

Comment:
Can we add a plugin to this test that has missing optional dependencies to validate that this log does not disable or log optional dependencies

Reply:
5d1b4e5

I also added some for nested/indirect dependencies.